magefiles/tests.go

```diff
@@ -52,12 +52,20 @@ func Tests() error {
 		return err
 	}
 
-	err = dockerRun("run", "-d", "--name=redis", docker_Net, "-p=6379:6379", "redis:6.2.6")
+	if len(docker_Net) > 0 {
```

Comment:
This is a bit of an ugly fix for running tests on Mac OS. For Mac OS docker net is empty string which breaks the `dockerRun` call.